### PR TITLE
Add note in documentation for pow function

### PR DESCRIPTION
**Pull request by pradeepbhadani**
Add note to mention `pow` function accepts float values.
I missed commit in #14598

### CODE REVIEW

File: website/source/docs/configuration/interpolation.html.md [hunk 1]
```diff
@@ -314,6 +314,8 @@ The supported built-in functions are:
 
   * `pow(x, y)` - Returns the base `x` of exponential `y`.
```
Comment (grubernaut) [inline website/source/docs/configuration/interpolation.html.md]: Remove the note from the line below, and change this line to be: 
```
* `pow(x, y)`  - Returns the base `x` of exponential `y` as a float
```

File: website/source/docs/configuration/interpolation.html.md [hunk 2]
```diff
@@ -312,12 +312,13 @@ The supported built-in functions are:
...
     This will create a plan diff between two different hosts, unless the filepaths are the same.
 
-  * `pow(x, y)` - Returns the base `x` of exponential `y`.
+  * `pow(x, y)`  - Returns the base `x` of exponential `y` as a float.
 
     Example:
     * `${pow(3,2)}` = 9
     * `${pow(4,0)}` = 1
 
+
```
Comment (grubernaut) [inline website/source/docs/configuration/interpolation.html.md]: Extra newline added

Comment (grubernaut) [inline website/source/docs/configuration/interpolation.html.md]: Extra newline added

Comment (pradeepbhadani) [inline website/source/docs/configuration/interpolation.html.md]: ahh..my markdown renderer was asking to add one more line to have separation with the following point.

Removed extra line

File: website/source/docs/configuration/interpolation.html.md [hunk 3]
```diff
@@ -312,7 +312,7 @@ The supported built-in functions are:
   * `pathexpand(string)` - Returns a filepath string with `~` expanded to the home directory. Note:
     This will create a plan diff between two different hosts, unless the filepaths are the same.
 
-  * `pow(x, y)` - Returns the base `x` of exponential `y`.
+  * `pow(x, y)`  - Returns the base `x` of exponential `y` as a float.
```
Comment (grubernaut) [inline website/source/docs/configuration/interpolation.html.md]: Remove the extra space before `- Returns`

Comment (pradeepbhadani) [inline website/source/docs/configuration/interpolation.html.md]: Removed

**Pull request merged**